GoToProject/android/app/src/main/AndroidManifest.xml

```diff
@@ -10,6 +10,10 @@
       android:roundIcon="@mipmap/ic_launcher_round"
       android:allowBackup="false"
       android:theme="@style/AppTheme">
+      <meta-data
+         android:name="com.google.android.geo.API_KEY"
+         android:value="AIzaSyB4SwDlMm4Dr2c7j5A-iRbFbAWxIKO-tOA"/>
```

Comment:
Is there a way you can hide your API Key from this section?

Reply:
checking

Reply:
API key has been removed from visible files, I created and env file out of git